sm/plugins/tedge_docker_plugin/tedge_docker_plugin.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
```

Comment:
My comment is about the file name. SM Agent will report the file name as module type. So, the module type will be `tedge_docker_plugin.sh`. I don't  think we want it. Should we rename the file `docker` instead?

Reply:
Yeah, for now I've just kept it consistent with the rest of the plugins. I was hoping that the rename will happen as part of the "installation" of this plugin, for eg: during the debian package installation. But, it depends on how this plugin will be installed. So, I thought I'll defer this until the delivery model is finalized.